prime-router/metadata/hl7_mapping/ORU_R01/base/specimen.yml

```diff
@@ -145,5 +145,5 @@ elements:
     constants:
       hl7ObservationPath: '/PATIENT_RESULT/ORDER_OBSERVATION(%{orderIndex})/SPECIMEN(%{specimenIndex})/SPECIMEN_OBSERVATION'
       hl7OBXField: '/PATIENT_RESULT/ORDER_OBSERVATION(%{orderIndex})/SPECIMEN(%{specimenIndex})/SPECIMEN_OBSERVATION(%{specimenObservationIndex})/OBX'
-      observation: '%diagnostic.result[%resultIndex].resolve()'
```

Comment:
@mkalish This observation seemed to not be needed at all, maybe that was just coincidental because of the fields that were populated in the data. In cases where we did need `%observation`, I think the commented out value might be more accurate, does that seem right to you?

Reply:
💯 This is actually a great forcing example of why the current approach can't scale